src/App.css

```diff
@@ -67,17 +67,20 @@ h1 {
 h2 {
   font-size: 20px;
   color: #1c1a19;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
If you want all body copy to have this font, you can just apply it to `body` or `html`; you can specify exceptions (like display fonts) on the respective classes. Then, you can reduce the duplication across different elements. Not a huge deal though!